cloud/amazon/ec2_vpc_route_table.py

```diff
@@ -406,7 +413,6 @@ def ensure_route_table_absent(connection, module):
     route_table_id = module.params.get('route_table_id')
     tags = module.params.get('tags')
     vpc_id = module.params.get('vpc_id')
-    check_mode = module.params.get('check_mode')
```

Comment:
I ran into issues with this local variable in my testing.  This was several months ago though so I'm not entirely clear on why.  By using `module.check_mode` instead of `check_mode` my problems disappeared.  This seems like a common pattern in other modules so I'd like to stick with this change.